constraint_utils/constraint_basics.py

```diff
@@ -54,11 +54,33 @@ def load_all_possible_summary(filtered: bool = True) -> Dict[hl.Struct, int]:
 
 
 def load_tx_expression_data(context=True):
+    """
+    Load tx MatrixTable with processed expression data.
+
+    :param context: Whether the tx Table will be used by context Table. defaults to True.
+    :return: tx Table with expression data.
+    """
     tx_ht_file = 'gs://gnomad-public/papers/2019-tx-annotation/pre_computed/all.possible.snvs.tx_annotated.021819.ht' if context \
```

Comment:
Since there are only standard model and worst_csq model these two options as for now, this function may not be used